stories/Typography.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Typeset } from '@storybook/addon-docs';
+
+<Meta title="Base/Typography" />
+
+export const fontSizeBase = 16;
+export const fontFamilySystem = '-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, "Helvetica Neue", Helvetica, sans-serif';
+export const fontFamilyDisplay = `"Aeonik", ${fontFamilySystem}`;
+export const fontFamilyBody = `"Maison Neue", ${fontFamilySystem}`;
```

Comment:
Same goes for all of this – ideally storybook is rendering using shared configuration.